packages/@atjson/source-gdocs-paste/src/converter.ts

```diff
@@ -111,51 +111,40 @@ GDocsSource.defineConverterTo(OffsetSource, doc => {
       };
     })
     .forEach(({ start, end }) => {
-      // Convert single new lines to LineBreaks
-      doc.match(/(^|[^\n])\n([^\n]|$)/g, start, end).forEach(match => {
-        let newlineStart = match.start + match.matches[1].length;
-        let newlineEnd = match.end - match.matches[2].length;
+      // Multiple newlines indicate paragraph boundaries within the block boundary
+      let paragraphBoundaries = doc.match(/\n{2,}/g, start, end);
```

Comment:
This is probably a more performant regex and this makes the code easier to grok overall.